dev/com.ibm.ws.microprofile.config/src/com/ibm/ws/microprofile/config/interfaces/ConfigConstants.java

```diff
@@ -40,5 +40,6 @@
     public static final long DEFAULT_DYNAMIC_REFRESH_INTERVAL = 500;
     //The minimum refresh rate for dynamic sources ... if the refresh rate is set lower than this then it will be overridden by this value
     public static final long MINIMUM_DYNAMIC_REFRESH_INTERVAL = 500;
-
+    //The regex to specify allowable characters in an Environment Variable Config Source
+    public static final String ALLOWABLE_CHARS_IN_ENV_VAR_SOURCE = "[^A-Za-z0-9_]";
```

Comment:
while creating a separate constants file for 1.3 might be over-kill, the comment should probably make it clear that it is only used in 1.3